tests/testsuite.src/syn_copy.at

```diff
@@ -951,8 +951,6 @@ AT_DATA([prog.cob], [
        REPLACE OFF.
 ])
 
-AT_CHECK([$COMPILE_ONLY prog.cob], [1], [],
-[prog.cob:10: error: 'VAR-COMMA' is not defined
-])
+AT_CHECK([$COMPILE_ONLY prog.cob], [0], [], [])
 
 AT_CLEANUP
```

Comment:
as mentioned in the first commit's review: please duplicate that for "in preprocess listing" (or consider including that in the same test, after all it just writes to an _additional_ file which you can have a reference here, then just add `-P preproc.list` and compare the file afterwards.